src/cargo/sources/directory.rs

```diff
@@ -112,18 +113,19 @@ impl<'cfg> Source for DirectorySource<'cfg> {
             let pkg = src.root_package()?;
 
             let cksum_file = path.join(".cargo-checksum.json");
-            let cksum = paths::read(&path.join(cksum_file)).chain_error(|| {
-                human(format!("failed to load checksum `.cargo-checksum.json` \
-                               of {} v{}",
-                              pkg.package_id().name(),
-                              pkg.package_id().version()))
+            let cksum = paths::read(&path.join(cksum_file)).chain_err(|| {
+                format!("failed to load checksum `.cargo-checksum.json` \
+                         of {} v{}",
+                        pkg.package_id().name(),
+                        pkg.package_id().version())
 
             })?;
-            let cksum: Checksum = serde_json::from_str(&cksum).chain_error(|| {
-                human(format!("failed to decode `.cargo-checksum.json` of \
-                               {} v{}",
-                              pkg.package_id().name(),
-                              pkg.package_id().version()))
+            let cksum: Checksum = serde_json::from_str(&cksum)
+                .chain_err(|| {
```

Comment:
Could this call to `chain_err` stay on the previous line? As-is the indentation looks a little odd